test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -181,7 +181,64 @@ public void execute_addDuplicate_notAllowed() throws Exception {
                 expectedAB,
                 false,
                 Collections.emptyList());
+    }
+
+    @Test
+    public void execute_edit_invalidArgsFormat() throws Exception {
```

Comment:
your test names do not follow the convention specified in our coding guidelines. 
please take note and name them appropriately in your project